src/QUESTIONS.md

```diff
@@ -246,6 +269,17 @@ You can list all available software on CheriBSD with `pkg64c rquery %n` and
 Alternatively, you can search for a specific package using
 `pkg64c search <pattern>` and `pkg64 search <pattern>`.
 
+If you simply require a list of packages independently of CheriBSD, there are browsable HTML lists of the ports for both architectures: [AArch64](https://pkg.cheribsd.org/CheriBSD:20220828:aarch64.html) (hybrid) and [AArch64c](https://pkg.cheribsd.org/CheriBSD:20220828:aarch64c.html) (CHERI). In addition, there are tarballs on [pkg.cheribsd.org](https://pkg.cheribsd.org) that contain the raw JSON-formatted package data. To generate a list of CHERI packages with specific fields, you can use `curl`, `tar`, and `jq`:
```

Comment:
```suggestion
Without a CheriBSD host, you can browse CheriABI and hybrid ABI package repositories at [pkg.CheriBSD.org](https://pkg.CheriBSD.org/).

In addition, there are tarballs on [pkg.cheribsd.org](https://pkg.cheribsd.org) that contain the raw JSON-formatted package data. To generate a list of CHERI packages with specific fields, you can use `curl`, `tar`, and `jq`:
```

Reply:
Note that `CheriBSD:20220828:aarch64{,c}.html` is not a page for the current package repositories but package repositories for this specific ABI (`CheriBSD:20220828:*`). It will change with the next release and hence the description should not be specific to a package repository for a particular ABI.

Also, the ABIs are called "the hybrid ABI" and "CheriABI" (not CHERI).